app/controllers/trips_controller.rb

```diff
@@ -20,6 +20,12 @@ def create
         end
     end
 
+    def destroy
+        trip = Trip.find(params[:id])
+        trip.destroy if trip.valid?
```

Comment:
Alright, so this was my mistake.  You all don't need to check if valid for delete.